health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/CreateQuestionContext.js

```
@@ -38,6 +38,19 @@ const CreateQuestionContext = ({ onSelect, ...props }) => {
             return [...action.payload];
           }
         else return state;
+      case "CLEAR_DATA":
+        const newState = state.map(item => {
+          // Set isActive to false for all previous items
+          return { ...item, isActive: false };
+        });
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)